devops/dsa/README.rst

```diff
@@ -32,7 +36,7 @@ To start the Digital Slide Archive::
 
 This uses your current user id so that database files, logs, assetstore files, and temporary files are owned by the current user.  If you omit setting ``DSA_USER``, files may be created owned by root.
 
-Note that this example does not add any default tasks or sample files.  By default, it creates an ``admin`` user with a password of ``password``.  You can log in with the admin user and use the Slicer CLI Web plugin settings to add default tasks (e.g., ``dsarchive/histomicstk:latest``).
+The girder instance can now be accessed at http://localhost:8080 by default, it creates an ``admin`` user with a password of ``password``. Note that this example does not add any default tasks or sample files.  You can log in with the admin user and use the Slicer CLI Web plugin settings to add default tasks (e.g., ``dsarchive/histomicstk:latest``).
```

Comment:
I think this should either have a semi color or be separate sentences:
`The girder instance can now be accessed at http://localhost:8080 by default` -> `The girder instance can now be accessed at http://localhost:8080. By default`